next-i18next.config.js

```diff
@@ -3,8 +3,8 @@ const path = require('path');
 
 module.exports = {
   i18n: {
-    defaultLocale: 'en-US',
-    locales: ['en-US', 'de-DE'],
+    defaultLocale: 'en',
```

Comment:
What's the reason you've changed the locale in this way? Can you continue to use i18n defaults?

Reply:
In contentful we have also locale 'nl' and 'en'. I've changed it here also to 'nl' and 'en' so it stays uniform.
Also it looks more pleasing to use amexio.fr/nl/pricing instead of amexio.fr/nl-BE/pricing .
What do you mean with i18n defaults?